src/mscorlib/src/System/TimeZoneInfo.cs

```diff
@@ -29,6 +29,7 @@ internal enum TimeZoneInfoOptions
     };
 
     [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public sealed partial class TimeZoneInfo : IEquatable<TimeZoneInfo>, ISerializable, IDeserializationCallback
```

Comment:
`TimeZoneInfo` was originally in System.Core in .NET 3.5, but was type-forwarded to mscorlib in .NET 4.0. In the [reference source](https://github.com/Microsoft/referencesource/blob/90b323fe52bec428fe4bd5f007e9ead6b265d553/mscorlib/system/timezoneinfo.cs#L57), it has `[TypeForwardedFrom("System.Core, Version=3.5.0.0, Culture=Neutral, PublicKeyToken=b77a5c561934e089")]`. Does that need to match here?